bcipy/helpers/parameters.py

```diff
@@ -0,0 +1,186 @@
+"""Module for functionality related to system configuration"""
+from typing import Tuple
+from codecs import open as codecsopen
+from collections import abc
+from json import dump, load
+from pathlib import Path
+
+DEFAULT_PARAMETERS_PATH = 'bcipy/parameters/parameters.json'
+
+
+class Parameters(dict):
```

Comment:
This makes me really happy. I like the parameters being a class that can handle itself.